src/command/helpers/prompt.js

```diff
@@ -11,13 +12,17 @@ async function prompt(message, choices, options = {}) {
   return list
 }
 
-prompt.input = async function(message) {
-  const { input } = await inquirer.prompt({
-    type: 'input',
-    name: 'input',
-    message,
+prompt.input = function(message) {
```

Comment:
I didn't like the other input because it would show up like
```
... Some really long question
?:
```
even if we kept it on the same line, it would look like
```
? .. Some really long question...:
```

It felt wrong so now this makes it look like
```
  Some really really long question?
```